theme/less/homepage.less

```diff
@@ -23,10 +23,16 @@ main article
 main article ol
 {
   margin-top: 0;
+  line-height: 1.5rem;
+  align-self: center;
 }
+
 main article h1
 {
-  margin: 6px 0 6px 0;
+  margin: 15px 0 6px 0;
+  font-weight: 300;
+  font-size: 35px;
+  color: black;
```

Comment:
I think we can here define black color as `color: #181818` for consistency.

Reply:
Good catch, agree.